examples/access-control-cedar/notebooks/01-Bootstrap.ipynb

```
@@ -0,0 +1,229 @@
+    "\n",
+    "CATALOG_URL = \"http://lakekeeper:8181/catalog\"\n",
+    "MANAGEMENT_URL = \"http://lakekeeper:8181/management\"\n",
+    "KEYCLOAK_TOKEN_URL = \"http://keycloak:8080/realms/iceberg/protocol/openid-connect/token\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Bootstraping Lakekeeper\n",
+    "This Notebook performs bootstrapping via python requests. It only works if the server hasn't previously bootstrapped using the UI!"
```

Ruff (0.14.14)

[error] Probable use of `requests` call without timeout

(S113)